superbench/benchmarks/micro_benchmarks/kernel_launch_overhead/CMakeLists.txt

```diff
@@ -0,0 +1,38 @@
+# Copyright (c) Microsoft Corporation.
```

Comment:
Do we have some reference for this? I think it's a general CMakeLists and maybe we can add a reference.

Reply:
Yes, I also think it has some common lines, like the ARCHS settings. I referred the cmake file of cutlass. I can extract them into one separated file.